packages/map-components/templates/react/src/index.tsx

```diff
@@ -2,6 +2,10 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 
 import { ArcgisMap, ArcgisSearch, ArcgisLegend } from "@arcgis/map-components-react";
+// import defineCustomElements to register custom elements with the custom elements registry
+import { defineCustomElements } from "@arcgis/map-components/dist/loader";
+// Register custom elements
+defineCustomElements();
 
 import "@arcgis/core/assets/esri/themes/dark/main.css";
```

Comment:
Hmmm, double check the CSS. It might need to be the CDN and not pulled from node_modules.

Reply:
Works OK

Reply:
Yep it will work, but for consistency the CSS should be using CDN here.

Reply:
Sorry, I should have given a link. Here's the pattern we've been using for ESM samples: https://github.com/Esri/jsapi-resources/blob/main/esm-samples/jsapi-react/src/index.css#L1

```css
@import 'https://js.arcgis.com/<version>/@arcgis/core/assets/esri/themes/light/main.css';
```